llvm/include/llvm/CodeGen/LiveDebugVariables.h

```diff
@@ -21,23 +21,25 @@
 #define LLVM_CODEGEN_LIVEDEBUGVARIABLES_H
 
 #include "llvm/CodeGen/MachineFunctionPass.h"
+#include "llvm/IR/PassManager.h"
 #include "llvm/Support/Compiler.h"
+#include <memory>
 
 namespace llvm {
 
 template <typename T> class ArrayRef;
 class LiveIntervals;
 class VirtRegMap;
+class LDVImpl;
```

Comment:
Instead of moving this into llvm namespace, could declare inside the class?

Reply:
Works